src/test/java/gov/healthit/chpl/aqa/asserts/ViewProductPageAsserts.java

```diff
@@ -0,0 +1,83 @@
+package gov.healthit.chpl.aqa.asserts;
+
+import static org.junit.Assert.assertTrue;
+import static org.testng.Assert.assertEquals;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchElementException;
+
+import cucumber.api.java.en.Then;
+import gov.healthit.chpl.aqa.pageObjects.ViewProductPage;
+import gov.healthit.chpl.aqa.stepDefinitions.Base;
+
+/**
+ * Class ViewProductPageAsserts definition.
+ */
+public class ViewProductPageAsserts extends Base {
+
+    /**
+     * Assert that product edit link exists on view version page.
+     * @param productId is the product id passed in page objects
+     */
+    @Then("^I see product edit link for product \"([^\"]*)\"$")
+    public void iSeeProductEditLink(final String productId) {
+        try {
+            ViewProductPage.editProductLink(getDriver(), productId).isDisplayed();
+            assertTrue(true);
+        } catch (NoSuchElementException e) {
+            assertTrue("Unable to find product edit link", false);
+        }
+    }
+
+    /**
+     * Assert that product split link exists on view version page.
+     * @param productId is the product id passed in page objects
+     */
+    @Then("^I see product split link for product \"([^\"]*)\"$")
+    public void iSeeProductSplitLink(final String productId) {
+        try {
+            ViewProductPage.splitProductLink(getDriver(), productId).isDisplayed();
+            assertTrue(true);
```

Comment:
If you put the previous line inside the assertTrue then this would make sense

Reply:
or assertTrue((ViewProductPage.splitProductLink(getDriver(), productId).isDisplayed()));